setup.py

```diff
@@ -35,7 +35,8 @@
         'requests-toolbelt==0.8.0',
         'requests==2.20.1',
         'scipy==1.0.0rc1',
-        'typing==3.6.4'
+        'typing==3.6.4',
+        'six',
```

Comment:
please add a six version or a comment in the file explaining why it doesn't have a version